src/test/it/jaxwscommons-81/pom.xml

```diff
@@ -51,7 +58,6 @@
                             <goal>wsgen</goal>
                         </goals>
                         <configuration>
-                            <executable>${tool.wsgen}</executable>
```

Comment:
These changes to this test case worries me a bit. The issue at java.net is no longer available, but the summary for the issue says "Allow 'fork'" (see https://www.mojohaus.org/jaxws-maven-plugin/release-notes-221.html). So possibly the usage of 'executable' is what this test case is all about.

Reply:
If you only have Java 11+ on your machine, this test fails because the commands wsgen and wsimport don't exist anymore. Using adoptopenjdk-12.0.2 the following error appears in target/it\ tests/jaxwscommons-81/build.log:
`(...)
[INFO] jaxws:wsimport args: [-keep, -s, '/Users/thorsten/git/theit-jaxws-maven-plugin/target/it tests/jaxwscommons-81/target/generated-sources/wsimport', -d, '/Users/thorsten/git/theit-jaxws-maven-plugin/target/it tests/jaxwscommons-81/target/classes', -Xnocompile, "file:/Users/thorsten/git/theit-jaxws-maven-plugin/target/it%20tests/jaxwscommons-81/src/wsdl/HelloWs.wsdl"]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.783 s
[INFO] Finished at: 2019-09-11T21:39:20+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:jaxws-maven-plugin:2.6-SNAPSHOT:wsimport (id4) on project jaxwscommons81: Cannot execute: /Library/Java/JavaVirtualMachines/adoptopenjdk-12.jdk/Contents/Home/../bin/wsimport -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.codehaus.mojo:jaxws-maven-plugin:2.6-SNAPSHOT:wsimport (id4) on project jaxwscommons81: Cannot execute: /Library/Java/JavaVirtualMachines/adoptopenjdk-12.jdk/Contents/Home/../bin/wsimport
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
(...)`

Reply:
I think we should then keep the test as it was and configure it to only run with Java 8. Preferrably the release is performed with Java 8 and then the test is executed as intended.